src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 300ms ease;
```

Comment:
Consider using a variable for the transition duration (300ms) to maintain consistency across the project. Example: `$transition-duration: 300ms;` 🌟